WordPress/Classes/ViewRelated/Plans/PlanDetailViewController.swift

```diff
@@ -96,6 +96,8 @@ class PlanDetailViewController: UIViewController {
         planTitleLabel.text = plan.fullTitle
         planDescriptionLabel.text = plan.description
         planPriceLabel.text = priceDescriptionForPlan(plan)
+
+        purchaseButton.alpha = (plan == .Free) ? 0 : 1
```

Comment:
`hidden` makes things a little tricky because the button is in a stack view with the labels – if you hide it, the stack view redistributes the available space so that the labels are spaced further apart. I can probably stick in some extra constraints to handle this though.